dataikuapi/dss/ml.py

```diff
@@ -253,6 +253,50 @@ def get_algorithm_settings(self, algorithm_name):
 
         return self.mltask_settings["modeling"][algorithm_name.lower()]
 
+    def get_hints_settings(self):
+        """
+        Gets the hints settings for a mltask. This returns a reference to the
+        hints' settings, not a copy, so changes made to the returned object will be reflected when saving.
+
+        This method returns a dictionary of the settings with:
+        - 'enabled': indicates if the hints are enabled globally, if False, all hints will be disabled
+        - 'settings': a dict comprised of:
+          - 'type': the hint type
+          - 'enabled': indicates if the hint type is enabled, if False, all hints of that type will be disabled
```

Comment:
If it's a list of dict or a dict (what's the key then?)

Reply:
Clarified: a list of dict